src/node_mgmt/commands.py

```diff
@@ -55,26 +56,23 @@ def env_scan(node):
     scan_result = scanner.do_scan()
     logger.info('Completed environment scan. Submitting results to API and MQTT')
     node.api.post_env_scan(scan_result)
-    if node.mqtt_client.publish(scan_result, topic=MQTT_STATE_TOPIC):
```

Comment:
Although this MQTT push isn't actively used currently, there's strictly speaking no reason to remove it, and it can be handy to keep. It will likely be used in the future - so would prefer to leave it in for now unless there's a reason to remove it that I'm not aware of of course.